MANIFEST.in

```diff
@@ -1,8 +1,17 @@
 # *********
 # |docname|
 # *********
-graft runestone
+graft runestone/dist
 include requirements.txt
```

Comment:
Wow, except for the first three lines, this file has not been touched in 6+ years

Doesn't this exclude a bunch of import .py files though?? `runestone/activecode/activecode.py` and many others?

Reply:
No, on line 42 (`packages=find_packages(exclude=["*.*.test"]),`) [find_packages](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#using-find-or-find-packages) finds and includes all the Python files in the Runestone Components. 

Also, I built and tested all the Runestone Components in the overview book using the distribution produced by this setup.py and everything worked.

Here's a snippet from running `python setup.py sdist`:

```
copying runestone\assignment\__init__.py -> runestone-6.0.4\runestone\assignment
copying runestone\blockly\__init__.py -> runestone-6.0.4\runestone\blockly
copying runestone\blockly\blockly.py -> runestone-6.0.4\runestone\blockly
copying runestone\cellbotics\__init__.py -> runestone-6.0.4\runestone\cellbotics
copying runestone\chapterdb\__init__.py -> runestone-6.0.4\runestone\chapterdb
copying runestone\chapterdb\dbchapterinfo.py -> runestone-6.0.4\runestone\chapterdb
copying runestone\clickableArea\__init__.py -> runestone-6.0.4\runestone\clickableArea
copying runestone\clickableArea\clickable.py -> runestone-6.0.4\runestone\clickableArea
copying runestone\codelens\__init__.py -> runestone-6.0.4\runestone\codelens
copying runestone\codelens\callback_module.py -> runestone-6.0.4\runestone\codelens
copying runestone\codelens\htmlFrame.py -> runestone-6.0.4\runestone\codelens
```